java/Ice/greeter/README.md

```diff
@@ -12,64 +12,32 @@ The same client works with both.
 
 This project uses Gradle and the [Application plugin]. To build the client and server applications, run:
 
-### Linux/macOS
-
 ```shell
 ./gradlew installDist
 ```
 
-### Windows
-
-```shell
-gradlew installDist
-```
-
 This will compile each project and install the distributions in the `build/install` subdirectory of each project.
 
 ## Running the Servers
 
 First, start one of the server implementations:
 
-### Linux/macOS
-
-```shell
-./server/build/install/server/bin/server
-```
-
-### Windows
-
 ```shell
-server\build\install\server\bin\server
+./gradlew :server:run --quiet
```

Comment:
The title should be 'Running the Server'. You should only run one server at a time.

Reply:
Fair enough, I made it singular.